doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -351,27 +443,54 @@ Unary statements are composed of a unary operator (either `+`or `-`) and are use
  
 ## Geographical Queries
 
-Geographical queries are specified by means of the `geometry` and `coords` parameters.
+Geographical queries are specified by means of the `geometry` and `coords` parameters. 
 
 **geometry** defines a geografical area so that only those entities located in such area match the query.
 It is composed of a token list separated by ';'. The first token is the shape of the geometry,
-the rest of tokens (if any) depends on the shape. 
-
-In particular, the following geometries are supported:
-
-+ `geometry=circle;radius:4000`, inside a circle (radius is in meters).
-+ `geometry=circle;radius:4000;external`, outside a circle (radius is in meters).
-+ `geometry=polygon`, inside a polygon.
-+ `geometry=polygon;external`, outside a polygon.
-
-**coords** represents a a list of pairs of geographical coordinates (latitude, longitude) separated by ','.
-Every pair of coordinates represent a geographical point. The list length and its semantics depend on the shape
-specified by the **geometry** parameter:
-
-+ In the case of `circle` geometry, the parameter includes just one element, which
-  is the center of the circle.
-+ In the case of `polygon` includes a list of elements, which are the vertices of the polygon.
-
+the rest of tokens (if any) depends on the shape. The following geometries (see Simple Location Format)
+must be supported:
+
++ `geometry=point`, entities located at exactly that point on the Earth surface.
++ `geometry=line`, entities located along the polygonal line defined.
++ `geometry=circle;radius:4000`, entities inside a circle of the given radius (radius unit must be meters).
++ `geometry=circle;radius:4000;external`, entities outside a circle of the given radius (radius unit must be meters).
++ `geometry=polygon`, entities inside a polygon.
++ `geometry=polygon;external`, entities outside a polygon.
++ `geometry=box`, entities inside a bounding box.
++ `geometry=box;external`, entities outside a bounding box. 
```

Comment:
A richer set of modifier could be considered for circle, polygon and box, in particular:

* intersect: to mean intersection between the geometry defined by the query and the geometry in the entity attribute
* inside (or maybe it is better "within"?): to mean that the geometry in the entity attribute is inside the area defined by the query
* outside (instead of external): to mean that the gemoetry in the entity attribute is outside the area defined by the query

In addition, due to geo spacial query capabilities could be highly depedent on particular implementtation, the NGSIv2 API should implement a standard mechanism so if a given implementation is not able to solve a particular query type, an error could be reported to the client.